mev-boost-rs/src/service.rs

```diff
@@ -16,6 +16,7 @@ pub struct Config {
     pub host: Ipv4Addr,
     pub port: u16,
     pub relays: Vec<String>,
+    pub public_keys: Vec<String>,
```

Comment:
the format has the public key _in_ the URL which is already provided in the `relays` entry to this config; so we can remove this -- see the comment below about a way to grab the keys from the URLs themselves while making `Relay` instances